### ISSUE TITLE
Single quote ESI tags aren't matched

### ISSUE THREAD

**Issue by jloh** [pintsized/ledge#164]
Unsure whether this is intended or not but ESI tags with single quotes wont match the below.

```lua
    local src, err = ngx_re_match(
        include_tag,
        [[src="([^"]+)"]],
        "oj"
    )
```

https://github.com/pintsized/ledge/blob/master/lib/ledge/esi/processor_1_0.lua#L385

**Comment by hamishforbes** [comment]
Yeah the ESI spec doesn't specify but it's XML-based and XML allows for single or double quotes.
So I guess we should support both ideally.

**Comment by hamishforbes** [comment]
I think this is true for `<esi:when test="">` tags as well.

Perhaps a good solution would be an attribute parsing utility function that can return you a table of attribute / value pairs.
It should handle multiple attributes (something else we don't handle well at the moment) and this single/double quotes issue
